### DISCUSSION TITLE
Cannot launch fluent with working journal

### DISCUSSION THREAD

**Discussion by robcodesnew** [ansys/pyfluent#2230] [Q&A]
Hi all, 
I have been trying to launch fluent using a solve journal I have generated. My command to do so is the following:
`journal_path= "Z:/..../tmp/naca0015_aoa_0_journal_solve.jou"`
`cwd = "Z:/..../tmp/"`
`solver=pyfluent.launch_fluent(product_version = "22.2.0", 
                              version = "3d", 
                              processor_count = 12,
                              show_gui = True, 
                              journal_filepaths= journal_path, 
                              cwd=cwd)`

However this command gives me the following error:
pyfluent.launcher ERROR: Exception caught - RuntimeError
LaunchFluentError: 
Fluent Launch string: start "" "C:\Program Files\ANSYS Inc\v221\fluent\ntbin\win64\fluent.exe" 3ddp -t12  -sifile="C:\Users\...\AppData\Local\Temp\serverinfo-y641hgp0.txt" -nm -hidden

The command does open the GUI and identify the processors to be used, so I suspect the issue is with the journal filepath parameter. 
The journal itself is correct because the exact value of journal_path pasted into the GUI's "read journal" option runs the simulation perfectly. Could anyone help me identify my mistake here?

**Comment by mkundu1** [comment]
This is a bug with Fluent 22.2, we'll look into fixing it. As a workaround, pass the journal file as an additional argument with Fluent 22.2:
```
solver = pyfluent.launch_fluent(product_version="22.2.0", additional_arguments="-i a.jou")
```

**Reply by robcodesnew** [reply]
Thanks for the reply. I have another slight issue. When I run the code with `product version = "22.2.0"` it seems to launch Fluent 22R1. I have fluent 22R2 installed on my machine. Is there an explanation for that? 
![image](https://github.com/ansys/pyfluent/assets/150433762/17c142f7-e6ad-4b5c-bb1a-d1e90d547392)

**Reply by raph-luc** [reply]
@robcodesnew try checking the value of your `AWP_ROOT222` environment variable, see if it is pointing to the correct Ansys install. It should be set to something like `C:\Program Files\ANSYS Inc\v222` (depending on where your Ansys 2022 R2 is installed).

**Reply by robcodesnew** [reply]
Ok great! Thanks, that worked. One last question: what procedure would you recommend to launch a whole list of journals? Running the `launch_fluent()` multiple times? I was planning on using the journal_filepaths parameter but it seems I cannot input  a list like that that with additional_arguments

**Reply by mkundu1** [reply]
To read multiple journals using `additional_arguments`:
```
solver = pyfluent.launch_fluent(product_version="22.2.0", additional_arguments="-i a.jou -i b.jou")
```
